db/migrate/20190808084803_create_packages.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePackages < ActiveRecord::Migration[5.2]
+  def change
+    create_table :packages do |t|
+      t.string :package_name
+      t.string :company_name
+      t.integer :review_count
+      t.float :average_rating
```

Comment:
Who are you going to show the average_rating/date graph with this information?

Reply:
It's test search api(from seeds) (v0) for presentation